### ISSUE TITLE
[bug/enhancement]  "codespaces" domains added to counter.dev

### ISSUE THREAD

**Issue by Bellisario** [ihucos/counter.dev#105]
### Issue

Sometimes I use GitPod or GitHub Codespaces to develop: these services proxies local server via a temp subdomain. I just noted that a `*.gitpod.io` has been added to my "tracking" domains.

My suggestion is to prevent these "codespaces" platforms from being tracked because they're just for developing purposes and not for production.

### Workaround

Limit domains allowed from _Account Menu_, _Edit account_ then _Listed Domains_ section.

**Comment by ihucos** [comment]
Yes, there are some domains that we could ignore by default, agree.
